tests/pages/storage-page.ts

```diff
@@ -0,0 +1,28 @@
+import { expect, Locator, Page } from '@playwright/test';
+
+export class StoragePage {
+    readonly page: Page;
+    readonly encryptionSlider: Locator;
+    readonly encryptionEnabled: Locator;
+    readonly backButton: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.encryptionSlider = page.locator('#encryption-off');
+        this.backButton = page.getByRole('button', { name: 'Back' });
+        this.encryptionEnabled = page.getByRole('button', { name: 'Encryption settings' });
```

Comment:
should be grouped with the other state of the toggle above, with similar formatted name as suggested.
Basically it helps to visually locate the id in the order that more or less are in the UI.
Then we don't need an special locator for validation, but we would have two locator one for on one for off state.

Reply:
The problem is that this "Encryption settings" button has no text and only appears, if encryption is enabled. So the locator is as above. 
At the moment I have a version that uses the check() function for the checkbox, so we could think about a validation that checkes if the checkbox is really checked. WDYT?

Reply:
now is readable, we can see how this evolves, thanks for the change.